src/services/inplayer.account.service.ts

```diff
@@ -210,10 +211,15 @@ export const getCustomerConsents: GetCustomerConsents = async (payload) => {
 
 export const updateCustomerConsents: UpdateCustomerConsents = async (payload) => {
   try {
-    const { customer, consents } = payload;
+    const { customer, consents, consentValues } = payload;
```

Comment:
I'm confused as to why you added another field that has the same data (`consents` and `consentValues`.) Shouldn't it have been possible to just mutate the existing `consents` data?